src/UserList/component/UserListItem.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+
+function UserListItem({id, userName, userEmail, isSelected, onSelect}, props){
+    const style = isSelected ?
+        {
+            border:'2px solid green',
+            maxHeight:'50px',
+            maxWidth:'-webkit-fill-available'} : {};
+
+    const spanStyle = {
+        display:'flex',
```

Comment:
Use style property on components only in case when it is only available option. Static styling in JSX is bad pattern. It can have place only in case of dynamic styling. Create class name or use tag names as reference to element inside of css.